src/ethereum_test_tools/common/types.py

```diff
@@ -489,13 +503,71 @@ def to_kwargs_dict(account: "Dict | Account | None") -> Dict:
         return cls(**kwargs)
 
 
+class Sender(Address):
```

Comment:
I understand the intention, but this is just an EOA? The nonce even gets incremented by the `TransactionFixtureConverter` class in L1127 (https://github.com/ethereum/execution-spec-tests/pull/584/files#diff-5c36decc577af6b4761e4e8515d758992111a1d480e1266e5bd3b01ae875e184R1127), not in this class. So this could just be
```python
class EOA(Address):
```
?

Reply:
Changed the name of the class to `EOA`, and added the `get_nonce` function, which gets the current nonce and increases it by one at the same time (thought about calling it `get_increase_nonce` too, wdyt?).